frontend/cypress/integration/common/graph.ts

```diff
@@ -14,61 +14,73 @@ Then('user sees a minigraph', () => {
 
 Then('user sees the {string} namespace deployed across the east and west clusters', (namespace: string) => {
   cy.waitForReact();
-  cy.getReact('CytoscapeGraph')
+  cy.getReact('GraphPageComponent', { state: { graphData: { isLoading: false } } })
```

Comment:
Is there a way to make a custom command or something for this to make it easier to call? Like `cy.ensureGraphLoaded` or something?

Reply:
Probably.  We can look at that when the tests migrate from Cytoscape to PFT, sometime in the next couple of months.